src/test/py/bazel/runfiles_test.py

```diff
@@ -124,12 +136,15 @@ def testRunfilesLibrariesFindRunfilesWithoutEnvvars(self):
     self.AssertExitCode(exit_code, 0, stderr)
     bazel_bin = stdout[0]
 
-    exit_code, _, stderr = self.RunBazel(
-        ["build", "//bar:bar-py", "//bar:bar-java", "//bar:bar-sh"])
+    exit_code, _, stderr = self.RunBazel([
+        "build",
+        "--verbose_failures",
+        "--experimental_shortened_obj_file_path",
```

Comment:
Same here

Reply:
Same again: output path was too long and the test was failing.